server/Websocket/websocketserver.mjs

```diff
@@ -1,8 +1,11 @@
 import { createServer } from "http";
 import crypto from "crypto";
+import { send } from "process";
 
 const PORT = 8080;
 
+const rooms = new Map();
```

Comment:
This totally works for now, but you should think of persisting this to the database periodically later down the line, so that we do not lose data in case the server crashes or anything goes wrong.